apps/meteor/tests/e2e/administration.spec.ts

```diff
@@ -238,6 +240,52 @@ test.describe.parallel('administration', () => {
 		});
 	});
 
+	test.describe('Integrations', () => {
+		let messageCodeHighlightDefault: unknown;
+		const incomingIntegrationName = faker.string.uuid();
+
+		test.beforeAll(async ({ api }) => {
+			messageCodeHighlightDefault = await getSettingValueById(api, 'Message_Code_highlight');
+			await setSettingValueById(api, 'Message_Code_highlight', '')
+		});
+
+		test.beforeEach(async ({ page }) => {
+			await page.goto('/admin/integrations');
+		});
+
+		test.afterAll(async ({ api }) => {
+			await setSettingValueById(api, 'Message_Code_highlight', messageCodeHighlightDefault)
```

Comment:
Can we set the "real" default value here instead of the value before the executed test?